### DISCUSSION TITLE
About SoftSDKForUnity

### DISCUSSION THREAD

**Discussion by Lihao7788** [InfinyTech3D/SofaUnity#15] [Help - Setting up SofaUnity] [answered]
Hello, this is the problem I encountered when using SDK: 
my computer configuration is, GPU: NVIDIA GeForce RTX 2060,CPU:AMD Ryzen 5 3600 6-Core Processor ,
 Microsoft system is ,Windows 10 Pro
 Unity version is ,2021.3.32f1
SDK version is,SofaAPAPI-Unity3D_v22.12.00
Importing SDK into unity and replacing xu'ke'zheng1 will cause the following error when running the scene. In addition to this error, unity will crash from time to time.
Console Error  is
ExecutionEngineException: String conversion error: Illegal byte sequence encounted in the input.
SofaUnityAPI.SofaContextAPI.DisplayMessages () (at Assets/SofaUnity/Plugins/SofaUnityAPI/SofaContextAPI.cs:429)
SofaUnity.SofaContext.DoCatchSofaMessages () (at Assets/SofaUnity/Scripts/SofaContext.cs:564)
SofaUnity.SofaContext.ReconnectSofaScene () (at Assets/SofaUnity/Scripts/SofaContext.cs:618)
SofaUnity.SofaContext.Init () (at Assets/SofaUnity/Scripts/SofaContext.cs:405)
SofaUnity.SofaContext.StartSofa () (at Assets/SofaUnity/Scripts/SofaContext.cs:312)
SofaUnity.SofaContext.Awake () (at Assets/SofaUnity/Scripts/SofaContext.cs:258)

**Comment by epernod** [comment, marked as answer]
Hi,

sorry for this late reply.
This bug has been removed in the 23.06 and 23.12 version. Would you like to get a new trial license to test it?

Best regards
